spec/device_detector/bot_fixtures_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../spec_helper'
+
+describe DeviceDetector::Bot do
+
+  fixture_dir = File.expand_path('../../fixtures/parser', __FILE__)
+  fixture_files = Dir["#{fixture_dir}/bots.yml"]
+  fixture_files.each do |fixture_file|
+
+    describe File.basename(fixture_file) do
+
+      fixtures = YAML.load_file(fixture_file)
+
+      fixtures.each do |f|
+        user_agent = f["user_agent"]
+        bot = DeviceDetector::Bot.new(user_agent)
+
+        describe user_agent do
+          it "should be a bot" do
```

Comment:
In this project we have been using the "confident" expectation form, e.g. `it 'is a bot'`. Please change it if it's not much asking :)